src/root.css

```diff
@@ -0,0 +1,2 @@
+/* Temporary solution until Carbon is integrated into the reference app. */
+@import "https://unpkg.com/carbon-components/css/carbon-components.css";
```

Comment:
We might want to wait with merging until Carbon is in the app shell. As soon as that happened, this file can be removed (it's just a temporary hack to get the CSS working).